openeo/Sentinel-3.ipynb

```diff
@@ -1,5 +1,93 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "96b5d47a-3d98-41a1-a3f1-ee1e2e511677",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/opt/conda/envs/openeo/lib/libtiff.so\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "140164313301568"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from rasterio import logging\n",
+    "\n",
+    "log = logging.getLogger()\n",
+    "log.setLevel(logging.ERROR)\n",
+    "\n",
+    "import warnings\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "\n",
+    "# for tiffwarning:\n",
+    "import ctypes\n",
+    "from ctypes.util import find_library\n",
+    "\n",
+    "# Find the path to the library we want to modify\n",
+    "thePath = find_library('tiff')     # try \"gdal\" instead of \"tiff\" too\n",
+    "print(thePath)\n",
+    "\n",
+    "# Get handle to it\n",
+    "theLib = ctypes.CDLL(thePath)\n",
+    "theLib.TIFFSetWarningHandler.argtypes = [ctypes.c_void_p]\n",
+    "theLib.TIFFSetWarningHandler.restype = ctypes.c_void_p\n",
+    "theLib.TIFFSetWarningHandler(None)"
```

Comment:
What is all this logging/warning stuff for? Seems unnecessary (if not: describe the need for it in more detail).
Especially: what is all that TIFF stuff with `ctypes`? the notebook only uses netcdf as far as I can see

Reply:
Sorry. It was a wrong cell. Should be removed.